analyses/cell-type-wilms-tumor-06/00_run_workflow.sh

```diff
@@ -1,16 +1,28 @@
 #!/bin/bash
 
 # This script runs the module workflow.
+# There are two variables this script will take:
+# 1. The TESTING variable controls whether test data should be used.
+#  Setting TESTING=1 will use the test data.
+#  This variable is 0 by default.
```

Comment:
This doesn't seem accurate – it controls whether or not particular _steps are run._ What _data_ is being used is entirely controlled  by what happens to be symlinked at `data/current`, or am I missing something?

Reply:
No, you're not missing something - this description needs tweaking.